returnn/tensor/_tensor_extra.py

```diff
@@ -5,6 +5,7 @@
 
 from __future__ import annotations
 from typing import TYPE_CHECKING, Optional, Union, Type, Sequence, Tuple, List, Dict
+from numpy import number as numpy_number
```

Comment:
This is very uncommon. Just do `import numpy` instead, and then refer to it as `numpy.number`.